### PR TITLE
Test metadata

### PR DESCRIPTION
**Pull request by sg-wbi**
Add a test to verify that whatever `METADATA` are defined in `bigbio/utils/constants.py` are also defined in the dataloader script.

### CODE REVIEW

File: bigbio/utils/constants.py [hunk 1]
```diff
@@ -2,17 +2,15 @@
...
-    pairs_features,
-    qa_features,
-    text2text_features,
-    text_features,
-)
+from bigbio.utils.schemas import (entailment_features, kb_features,
+                                  pairs_features, qa_features,
+                                  text2text_features, text_features)
+
+METADATA = ["_LOCAL"]
```
Comment (galtay) [inline bigbio/utils/constants.py:9]: and `_LANGUAGES` now :)

Comment (hakunanatasha) [reply bigbio/utils/constants.py]: easy to add

**Review by hakunanatasha — DISMISSED** [review]
This is ready to merge @sg-wbi @galtay 

Maybe later, it would be interesting to have tabs on this with regards to changes in expected dataset size/number of entities etc based on time stamps. I'm curious for example to compare how this might relate to reported values in literature (i.e. some datasets do "add" more information over time).

File: bigbio/utils/constants.py (hunk 1 above)
Comment (hakunanatasha) [reply bigbio/utils/constants.py]: @galtay just added this

**Comment by hakunanatasha** [comment]
@sg-wbi @galtay some merge conflicts as the constants file adopted some new lines; I fixed these to maintain anything new added from `main`.

**Review by hakunanatasha — APPROVED** [review]

**Review by galtay — APPROVED** [review]

**Pull request merged**